.github/workflows/prefer-typescript.yml

```
@@ -2,9 +2,10 @@ name: Prefer TypeScript
 
 on:
   push:
-    branches-ignore:
-      - "dependabot/**"
+    branches:
+      - 'master'
   pull_request:
+    types: [synchronize, opened, reopened, ready_for_review]
```

Should we add a path here, eg:

```suggestion
    paths:
      - "superset-frontend/src/**"
    types: [synchronize, opened, reopened, ready_for_review]
```